test/integration/hasMany.test.js

```diff
@@ -409,7 +409,98 @@ describe('hasMany', function () {
       expect(project2.members, 'to have length', 4);
       expect(DataLoader.prototype.load, 'was not called');
     });
+  });
+
+  describe('deep association', function () {
```

Comment:
include.seperate just perhaps? as that's what it tests for at the moment

Reply:
Or "deep association with include.separate"? Because this only happens when it's a second level include.

Just say the word, I'll fix it right away.

Reply:
Ah yeah of course it only happens on deep includes as it will call `get()` again.
Call it `multiple levels of include.seperate` i suppose, not too important

Reply:
Already called it "deep association with include.separate" before your remark. Way to jump the gun, I guess... :)